template.rb

```diff
@@ -17,7 +17,8 @@ def apply_template!
   template "README.md.tt", force: true
   remove_file "README.rdoc"
 
-  template "example.env.tt"
+  template "example.env.tt", "example.env"
+  template "example.env", ".env"
```

Comment:
Does `.env` actually need to be templated? `bin/setup` copies it at the moment, so it might not need to be templated here as well?

Reply:
@joshmcarthur and I discussed this in person but for everybody playing along here, `.env` needs toe exist **before** `bin/setup` runs because I have used `ENV.fetch("SENTRY_ENV")` in the initializer.